wintaser/InputCapture.cpp

```diff
@@ -626,7 +626,7 @@ void InputCapture::ProcessInputs(CurrentInput* currentI, HWND hWnd){
 			continue;
 
 		// Now we build the SingleInput, and check if it's mapped to something.
-		SingleInput siPressed = {SINGLE_INPUT_DI_KEYBOARD, k, ""};
+		SingleInput siPressed = { SINGLE_INPUT_DI_KEYBOARD, static_cast<SHORT>(k), "" };

```

Comment:
Hold on. Shouldn't ´k´ be an `unsigned char`? It's also passed to ´IsModifier()´ below which takes an `unsigned char` as the parameter.